Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/StateSynchronization/StateSynchronizationObserver.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
     /// <summary>
     /// This class observes changes and updates content on a spectator device.
     /// </summary>
-    public class StateSynchronizationObserver : NetworkManager<StateSynchronizationObserver>, ICommandHandler
+    public class StateSynchronizationObserver : NetworkManager<StateSynchronizationObserver>
```

Comment:
>NetworkManager [](start = 48, length = 14)

It's starting to seem like NetworkManager should be its own true singleton. If somehow a StateSynchronizationObserver and StateSynchronizationBroadcaster get enabled, we'll start seeing multiple connections/disconnection events through the CommandRegistry

It also seems like the onconnected/ondisconnected calls in both the state synchronization broadcaster and state synchronization observer could be achieved by registering with a network manager singleton.

Reply:
I don't think NetworkManager can't be a singleton, as the HolographicCameraObserver and StateSynchronizationObserver are two NetworkManagers in the same Compositor scene.